### ISSUE TITLE
Don't divide loss by n_gradient_accumulation_steps

### ISSUE THREAD

**Issue by danbraunai-apollo** [ApolloResearch/e2e_sae#59]
We currently have the line:
```
loss = loss / n_gradient_accumulation_steps
```
in train_tlens_saes as well as other training scripts.

I'm not sure why this is here. It should be removed. Note that this may affect all of the current hyperparameters.
